README.md

```diff
@@ -1,2 +1,352 @@
+# Deep Diving 
+
+Before starting, I want to clear some of my notations. I will abide to the conventions of Linux man pages. In those:
+
+`[ ]` means that anything within (any values, information or parameters) are optional. You can chose one, more than one or none at all.
```

Comment:
> You can chose

You can *choose*

Reply:
> (any values, information or parameters)

(any values, information**,** or parameters

> anything within (any values, information or parameters) are optional

anything within (any value, information or parameter) *is* optional (anything is singular, so the parenthesis should be singular as well)

Reply:
Yes!! I should have take a step back and read the sentences again ...